SlimApp/ClientCard.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ClientCard.swift
+//  SlimApp
+//
+//  Created by Aleksandr Shapovalov on 10/01/23.
+//
+
+import SwiftUI
+
+struct ClientCard: View {
+    @State private var showingSheet = false
+    var name: String = "Antonio Ferrari"
+    var nextTraining: String = "Today"
+
+    var body: some View {
+        Button {
+            showingSheet.toggle()
+        } label: {
+            HStack(spacing: 20) {
+                VStack(alignment: .leading, spacing: 8) {
+                    Text(name)
+                        .font(.system(size: 24, weight: .bold))
+                    Text(nextTraining)
+                        .font(.system(size: 18, weight: .regular))
+                }
+
+                Spacer()
+                Divider()
+                Image("4aces")
+                    .resizable()
+                    .scaledToFit()
+            }
+            .padding(20)
+            .frame(maxWidth: .infinity, maxHeight: 110)
+            .background(.purple)
+            .foregroundColor(.white)
+            .mask(RoundedRectangle(cornerRadius: 28, style: .continuous))
+            .sheet(isPresented: $showingSheet) {
+                // MARK: Test ModalView, that will show a client info by clicking on ClientCard
+                TestModal()
```

Comment:
Since the modal is being created by @yukim17, maybe you should leave the sheet empty. This TestModal is probably going to end up being deleted. So leaving it empty might be the better option and it can be filled with the modal later

Reply:
i think that it it useful to see the behavior that we except from tapping this button, but if you suggest to remove this View and just have a blank space — I'm ok with that